source/SnapDump-Server/SDServer.class.st

```diff
@@ -27,6 +27,13 @@ SDServer class >> port: aNumber path: aFileReference [
 		yourself
 ]
 
+{ #category : #accessing }
+SDServer class >> repositoryDescription [
+	| snapDumpRepo |
+	snapDumpRepo := IceRepository registry detect: [ :each | each name = 'SnapDump' ].
```

Comment:
Now that is a dependency ;) And one I don't like too much. A mid-term goal is to switch to a minimal image and iceberg won't be included. Maybe there is another way to set the version?

Reply:
I just noticed the build uses iceberg. So why not. We need to find something better later